src/app/assets/Icons/BackArrowIcon.tsx

```diff
@@ -11,8 +11,8 @@ export default function BackArrowIcon(): JSX.Element {
       <path
         d="M14.942,26.906l-1.288,1.322a1.362,1.362,0,0,1-1.967,0L.406,16.659a1.449,1.449,0,0,1,0-2.018L11.686,3.067a1.362,1.362,0,0,1,1.967,0l1.288,1.322a1.457,1.457,0,0,1-.023,2.042L7.926,13.266H24.6A1.408,1.408,0,0,1,26,14.695V16.6A1.408,1.408,0,0,1,24.6,18.029H7.926l6.993,6.835A1.446,1.446,0,0,1,14.942,26.906Z"
         transform="translate(0.247 -2.497)"
-        fill="#707070"
-        stroke="#707070"
+        fill="--primary-border-color"
```

Comment:
I am not sure if this works. I think it should be
```suggestion
        fill="var(--primary-border-color)"
```

The problem here is, that you add an icon which is unused. Please don't do that because you can not make sure, that it works like intended.

Reply:
Thanks. I changed it. It works.